command-parser.js

```diff
@@ -30,6 +30,8 @@ const MESSAGE_COOLDOWN = 5 * 60 * 1000;
 
 const MAX_PARSE_RECURSION = 10;
 
+const VALID_COMMAND_TOKENS = ['/', '!'];
```

Comment:
I was going to say that support for `/` is hardcoded in several other places, but after grepping the codebase, apparently it isn't...